Algorithms/Problems/CoinChange/GreedyAlgorithm.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq.Expressions;
+using System.Security.Cryptography;
+using System.Text;
+
+public class GreedyAlgorithm
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6350770)